languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8789,20 +8789,6 @@ though, and has since been slightly modified)-->
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
       <rulegroup id="TOUTE_A_L_HEURE" name="toute a l’heure" default="temp_off">
-          <rule>
-              <pattern>
-                  <token>à</token>
-                  <marker>
-                      <token>toute</token>
-                  </marker>
-                  <token>à</token>
-                  <token>l'</token>
-                  <token>heure</token>
-              </pattern>
-              <message>Voulez-vous écrire <suggestion>tout</suggestion> ?</message>
-              <url>https://www.lalanguefrancaise.com/orthographe/a-tout-a-l-heure-ou-a-toute-a-l-heure/</url>
-              <example correction="tout">À <marker>toute</marker> à l'heure.</example>
-          </rule>
```

Comment:
I see that you removed this part (TOUTE_A_L_HEURE[1]) — why?

But the old rule (A_TOUTE_A_L_HEURE, lines 20269–20282) is still there. Was there a mix-up?

Reply:
Yes there was a mix-up on my part. I made the correct changes.